src/components/legend/draw.js

```diff
@@ -408,41 +410,98 @@ function setupTraceToggle(g, gd) {
         .attr('pointer-events', 'all')
         .call(Color.fill, 'rgba(0,0,0,0)');
 
-    traceToggle.on('click', function() {
+
+    traceToggle.on('mousedown', function() {
```

Comment:
Possibly - but this reminds me that we had better ensure that this change is compatible with the additional interactions the legend supports in `{editable: true}` config. For clarity, here's how that works currently and should continue to work:
- clicking on legend symbols toggles the trace (so doubleclicking them should isolate the trace)
- clicking on legend text, which outside of editable mode also toggles the trace, instead edits the text (so doubleclick should do nothing there)
- dragging **anywhere** in the legend - symbol, text, or background/border - moves the legend and does not toggle anything.

@rpaskowitz you can convert the plot you're testing to editable by calling:
```js
Plotly.newPlot(gd, gd.data, gd.layout, {editable: true})
```